hypothesis-python/docs/strategies.rst

```diff
@@ -201,5 +204,5 @@ opting out of the pytest plugin entirely.  Alternatively, you can ensure that Hy
 is loaded after coverage measurement is started by disabling the entrypoint, and
 loading our pytest plugin from your ``conftest.py`` instead::
 
-    echo "pytest_plugins = ['hypothesis.extra.pytestplugin']\n" > tests/conftest.py
+    echo "pytest_plugins = ['_hypothesis_pytestplugin']\n" > tests/conftest.py
```

Comment:
It feels a bit weird to have this module name in the documentation.

Would it make sense to also have a stub `hypothesis.extra.pytestplugin` module that just delegates to the real private-named one?